WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -31,6 +31,9 @@ typedef NS_ENUM(NSInteger, BlogListSections) {
 static CGFloat const BLVCHeaderViewLabelPadding = 10.0;
 static CGFloat const BLVCSiteRowHeight = 74.0;
 
+static int DisableAllMinSites = 10;
+static int DisableAllSitesThreshold = 6;
+static float DisableAllSitesInterval = 2.0f;
```

Comment:
We like to use framework types over c types. It would be nice to use `NSInteger` instead of `int` and `CGFloat` instead of `float`.  We also like to leave off the trailing `f` from CGFloat constants.  Doing so leaves it to the compiler to assign the correct "float" type.